.travis.yml

```diff
@@ -8,19 +8,19 @@ jobs:
     - stage: analyze_and_format
       name: "Analyze"
       os: linux
-      script: dartanalyzer --enable-experiment=non-nullable --fatal-warnings --fatal-infos .
```

Comment:
Migrate this back to trivial travis syntax, too?

Reply:
🤷‍♂️ I personally prefer this syntax because I can actually tell what the heck is going on lol